chart/values.yaml

```diff
@@ -119,3 +122,16 @@ ingress:
     nginx.ingress.kubernetes.io/backend-protocol: HTTPS
     nginx.ingress.kubernetes.io/ssl-passthrough: "true"
     nginx.ingress.kubernetes.io/ssl-redirect: "true"
+
+# Configure SecurityContext of the VCluster pod
+securityContext:
+  allowPrivilegeEscalation: false
+  # readOnlyRootFilesystem: true
```

Comment:
I meant to add a comment here^ that would link to #189,
readOnlyRootFilesystem: true doesn't work right now.

I'll add that comment if I'll be doing any follow-up fixes or changes.